### DISCUSSION TITLE
See reviewers of linked PRs

### DISCUSSION THREAD

**Discussion by ladislas** [community/community#10300] [Projects and Issues]
When linking a PR to an issue, we can only see who's in charge of the issue in assignees.

I would like to also see the reviewers of the linked PRs in the Reviewers column and be able to add/filter them as discussed here https://github.com/github/feedback/discussions/8164

**Comment by kevinwhoffman** [comment]
## Current Outcome

The Reviewers column added in https://github.com/github/feedback/discussions/8164 only displays reviewers if a pull request has been added directly to the project. However my team (and many others) only add issues to projects in order to avoid juggling both the issue and the pull request status in the same project. While we can see the linked PRs on the project board, we cannot see the reviewers requested in those PRs.

## Desired Outcome

The Reviewers column should display requested reviewers for issues that have linked pull requests.

![Google Chrome-WP Migrate-20220217T172253](https://user-images.githubusercontent.com/1938671/154536693-bf039fb8-75a9-43c9-b79a-021aaaf4584d.png)

**Comment by laymonage** [comment]
+1, would love to see this feature added! We tend to add issues and not the PRs to our Projects board, and being able to see the reviewers for the linked PRs would be very helpful.

**Comment by isanchez-ng** [comment]
I've been attempting to migrate from JIRA to GitHub Projects, aiming to visualize individuals' activities, whether they're acting as reviewers or assignees on pull requests or issues, within a RoadMap view. This visualization helps me assess their workload throughout the week. However, with the current features, the only option available is to visualize and group tasks by assignee. Unfortunately, I can't slice or group them by reviewer - check the image below -, which would greatly serve my purposes.

Upon reviewing this discussion, it appears that this limitation may exist because an issue can't have a designated reviewer. Therefore, I would like to request the feature suggested by @kevinwhoffman.

Currently, the only workaround is to create another "Reviewer Roadmap" view, where I manually adjust the project's filter to `is:open reviewers: @username`. However, this solution isn't scalable for teams with more than four software engineers. It requires manual filter edits, and it doesn't provide an overview of a developer's total workload, encompassing both reviewer and assignee tasks. This functionality closely resembles what the `is:open involves:username` filter offers within the issue or pull request filter bar. I would greatly appreciate having this capability integrated into the overall project view.

![image](https://github.com/community/community/assets/81649278/249b6389-92f3-4eee-be18-acd45722e04d) ![image](https://github.com/community/community/assets/81649278/ba13cf71-7444-48fe-9ae0-d9e0a43904e2)

**Comment by imaksimova** [comment]
+1 ^^^

Using the feature release board template, I’d like to identify which developer is responsible for reviewing the pull request (PR) linked to an issue. Each PR already has a designated reviewer, and the PR is properly linked to its corresponding issue.

Is there a way to automatically pull the reviewer from the PR and display them in the "Reviewer" column on the board? This works well when the PR itself appears on the board, but it would be helpful to also see who is responsible for the code review when only the issue is shown.

<img width="1172" alt="image" src="https://github.com/user-attachments/assets/7ed57115-7d0a-43b7-87a4-e281fccd59a4" />

**Comment by stherold** [comment]
Any progress here?
